tests/fixtures/operator/ssp.py

```
@@ -0,0 +1,35 @@
+
+@pytest.fixture()
+def ssp_resource_scope_function(admin_client, hco_namespace):
+    return get_ssp_resource(admin_client=admin_client, namespace=hco_namespace)
+
+
+@pytest.fixture(scope="class")
+def ssp_resource_scope_class(admin_client, hco_namespace):
+    return get_ssp_resource(admin_client=admin_client, namespace=hco_namespace)
+
+
+@pytest.fixture()
```

Ruff (0.15.21)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)